EMRM.py

```diff
@@ -151,12 +182,28 @@ def get_delta_x(self, word, lpos):
         return(delta_x)
         
 class OneFixation:
+    """ Class of a fixation dwelling at a location (in terms of characters) for a period of time.
+
+    Attributes:
+        lpos: A number indicating landing position in terms of characters.
+        fix_dur: A number indicating fixation duration.
+    """
     def __init__(self, lpos, fix_dur):
         self.lpos = lpos
         self.fix_dur = fix_dur
 
 class OneTrial:
+    """ Class of a trial of identifying a word.
+
+    Attributes:
+        reader: A OneVirtualReader with a vocabulary and some visual properties.
+        word: A string indicating the true word to be identified.
+        x: A vector indicating posterior distribution log-odds.
```

Comment:
somewhere, we'll need to explain what this representation is. Maybe the readme is best?

Reply:
added a graph in readme & cited papers, though not sure if we also need to include math equations in readme...

Reply:
ok, i think this is good enough now. But maybe it's just worth clarifying the description of x a little more to something more specific like "A vector with dimensionality equal to vocab size, with the ith element gives the log-odds of word i relative to word 0." Or something? (I'm not sure if what I wrote was exactly right given how you set it up?)